composer.json

```diff
@@ -34,7 +34,6 @@
     "require": {
         "laravel/framework": "^10.0",
         "prologue/alerts": "^1.0",
-        "backpack/theme-tabler": "dev-main as 1.0.0",
```

Comment:
This is what is preventing the tests from running @tabacitu 

It seems tabler was added as a dependency of backpack/crud in v6 branch ?

Reply:
Removed in https://github.com/Laravel-Backpack/CRUD/pull/5056 🎉 
In my env this made the unit tests run again. Actually on Github too, even though the PHPUnit v10 ones did not:
<img width="1271" alt="CleanShot 2023-05-03 at 09 49 48@2x" src="https://user-images.githubusercontent.com/1032474/235859656-1b613360-b72a-49c2-a004-845ae1e49080.png">

But that's a whole different issue I guess. We should probably remove PHPUnit v9 altogether... I don't see why we'd use both v9 and v10 at this point, since we only support L10 and PHP 8.1+